src/dispatcher.c

```diff
@@ -78,6 +78,12 @@
 #define check_avx512vbmi() (0)
 #endif
 
+#if defined(BUILD_SSE2_SIMDE)
+#define CHECK_SSE_LAST check_ssse2
```

Comment:
Is there a `check_ssse2` function?

Reply:
@mr-c we will have a base dependency for sse2, there will be no support for anything before SSE2. without SSE2 compilation will fail -for fat runtimes, one can use it for a non-fat runtime with SIMDE_BACKEND. We will fix this in another commit.